src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/Face/BaseTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using System;
+using System.IO;
+using System.Net.Http;
+using Microsoft.Azure.Test.HttpRecorder;
+using Face = Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+using Vision = Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+namespace FaceSDK.Tests
+{
+    public abstract class BaseTests
+    {
+        public static bool IsTestTenant = false;
+        private static readonly string FaceSubscriptionKey;
+        
+        static BaseTests()
+        {
+            // Retrieve the configuration information.
+            FaceSubscriptionKey = "";
+            Environment.SetEnvironmentVariable("AZURE_TEST_MODE", "Playback");
```

Comment:
@lebronJ you do not set this in test case, rather you set this in your connection string.
you use test framework to record tests
https://github.com/Azure/azure-powershell/blob/preview/documentation/Using-Azure-TestFramework.md#manually-set-environment-variables

Reply:
Removed this unnecessary setting
`Environment.SetEnvironmentVariable("AZURE_TEST_MODE", "Playback");`